src/lib.rs

```diff
@@ -4525,6 +4528,48 @@ pub unsafe trait FromBytes: FromZeros {
         }
     }
 
+    /// Reads a copy of `self` from the IO byte source `src`.
+    ///
+    /// This is useful for interfacing with operating system byte sinks (files,
+    /// sockets, etc.).
```

Comment:
IMO this is more specific than we need - non-OS resources can implement `io::Read` too. I think we can just remove this sentence.

Reply:
This is here solely for the benefit of SEO/Cmd+F. I don't say that these are the *only* uses for this method, but while non-OS resources *can* implement `io::Read`, you're almost *always* better off using our other methods in those cases.

It's not a load bearing sentence, but I think it's a helpful one.

Reply:
Ack.